src/main/java/soot/toDex/DexArrayInitDetector.java

```diff
@@ -117,6 +121,16 @@ public void constructArrayInitializations(Body body) {
     }
   }
 
+  /**
+   * Sets the minimum array length to consider
```

Comment:
"Only arrays with more than this number of elements will be identified, because we only need to handle those explicitly. For small arrays, we can have individual assignments for each element in the code, but for larger methods, this would exceed the allowed method size according to the JVM / Dalvik Spec".